sdk/inc/azure/iot/az_iot_common.h

```diff
@@ -45,10 +45,10 @@ enum az_result_iot
 enum az_log_classification_iot
 {
   AZ_LOG_MQTT_RECEIVED_TOPIC
-  = _az_LOG_MAKE_CLASSIFICATION(_az_FACILITY_IOT_MQTT, 1), ///< Accepted MQTT topic received.
```

Comment:
Breaking change: these are currently in-use. The classification is correct: this is IoT-specific MQTT usage (not generic).
These are used when the Hub/DPS clients parse the topics/payloads only.

Please revert changes in this file.